File: Smartsense.Backend/Smartsense.Entity/DTO/UserLoginRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Smartsense.Entity.DTO
{
   
    public class UserLoginRequest:BaseApiRequest
    {
        public string Email { get; set; }
        public string Password { get; set; }
        public bool isRemember { get; set; }
    }
}

File: Smartsense.Mobile.iOS/covid/covid/Api/Requests/UserLoginRequest.swift
//
//  UserLoginRequest.swift
//  covid
//
//  Created by SmartSense on 9.06.2021.
//

import Foundation

struct UserLoginRequest: BaseApiRequest, Encodable {
    private(set) var email: String?
    private(set) var password: String?
    private(set) var isRemember: Bool?
    var userId: Int64?
    
    init(email: String? = nil, password: String? = nil, isRemember: Bool? = nil) {
        self.email = email
        self.password = password
        self.isRemember = isRemember
    }
    
}

